File: 151003/Pavluchenko/publisher/src/main/java/by/bsuir/publisher/domain/Marker.java
package by.bsuir.publisher.domain;

import jakarta.persistence.Entity;
import jakarta.persistence.OneToMany;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

import java.util.List;

@Entity
@Table(name = "tbl_marker")
@SuperBuilder
@NoArgsConstructor
@Getter
@Setter
public class Marker extends BaseEntity {
    private String name;

    @OneToMany(mappedBy = "marker")
    private List<NewsMarker> newsMarkers;
}

File: 151003/Yagnnish/Lab1/Domain/Marker.cs
using System.ComponentModel.DataAnnotations;

namespace lab_1.Domain
{
    public class Marker:BaseEntity
    {
       
        private string _name;

        public Marker(long? id, string name):base(id) => _name = name;
        
        [StringLength(32,MinimumLength = 2)]
        public string Name { get => _name; } 
    }
}
